src/Http/Routing/src/ApiCompatBaseline.txt

```diff
@@ -0,0 +1,5 @@
+Compat issues with assembly Microsoft.AspNetCore.Routing:
+CannotChangeAttribute : Attribute 'System.AttributeUsageAttribute' on 'Microsoft.AspNetCore.Routing.HostAttribute' changed from '[AttributeUsageAttribute(68, AllowMultiple=false, Inherited=false)]' in the contract to '[AttributeUsageAttribute(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple=false, Inherited=false)]' in the implementation.
+MembersMustExist : Member 'Microsoft.AspNetCore.Routing.Matching.CandidateState.Values.set(Microsoft.AspNetCore.Routing.RouteValueDictionary)' does not exist in the implementation but it does exist in the contract.
```

Comment:
We should look into this one

Reply:
Values.Set is internal in the implementation:
https://github.com/aspnet/AspNetCore/blob/138e801e342af561c8f12c8bfd2591d47129db82/src/Http/Routing/src/Matching/CandidateState.cs#L53
But public in the ref assembly:
https://github.com/aspnet/AspNetCore/blob/138e801e342af561c8f12c8bfd2591d47129db82/src/Http/Routing/ref/Microsoft.AspNetCore.Routing.Manual.cs#L129
That seems like a bug in the ref assembly. It should be internal there too and use InternalsVisibleTo.

Reply:
@Pilchie does this example refer to your question? The tool picked up a diff between the `src` file and the corresponding `manual.cs` file

Reply:
This is not conclusive, since it was public on at least one side.  I'd want to see a case where it reports a mismatch in something that is internal on both sides.

Reply:
It's entirely possible we messed a few things up when creating and updating *.Manual.cs files e.g. did the all-important `internal` keyword get deleted accidentally? Suggest checking GenAPI output w/ `--all` because that tool uses the same base infrastructure as ApiCompat.